### PR TITLE
create timeStamp component

### LINKED ISSUE #408: Show age of transaction or time since mined
child of #387

### PR DESCRIPTION
**Pull request by artem-rozhkov-dev-optima**
closes #408

### CODE REVIEW

File: src/_designs/components/timeSince/index.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      let [, , time] = date.split(',')
+      date = date.replace(/,/g, '')
+      return this.isHeader ? `<div class="title">${title}</div> ${time}` : date
+    }
+  }
+}
+</script>
+<style scoped lang="scss">
+  @import "~@aeternity/aepp-components-3/src/styles/variables/colors";
+  @import '../../../style/mixins.scss';
```
[the remark this answers is not in the document]
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/timeSince/index.vue]: done

File: src/_designs/components/timeSince/index.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  .time-wrapper {
```
[the remark this answers is not in the document]
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/timeSince/index.vue]: done

File: src/_designs/components/timeSince/index.vue [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<template>
+  <div
+    class="time-wrapper"
+    :class="{ header: isHeader }"
+  >
+    <div
+      class="time-content"
+      v-html="getData(time)"
+    >
+      {{ getData(time) }}
```
[the remark this answers is not in the document]
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/timeSince/index.vue]: done

File: src/_designs/components/timeSince/index.vue [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  @import "~@aeternity/aepp-components-3/src/styles/variables/colors";
+  @import '../../../style/mixins.scss';
+  .time-wrapper {
+    /deep/ .title {
+      color: $color-neutral-negative-3;
+      @include font-size(m);
+    }
+
+    .time-content {
+      color: $color-neutral-negative-1;
```
[the remark this answers is not in the document]
Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/timeSince/index.vue]: done

File: src/_designs/components/timeStamp/index.vue [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    class="time-stamp"
+    :class="{ header: hasTitle }"
+  >
+    <AeText
```
Comment (emilsomm) [inline src/_designs/components/timeStamp/index.vue]: in this case we dont need the AeText component as you also set the same font styles. I would opt for a span instead

Comment (artem-rozhkov-dev-optima) [reply src/_designs/components/timeStamp/index.vue]: done

**Pull request merged**